packages/netlify-plugin-search-index/index.js

```diff
@@ -0,0 +1,108 @@
+const { tmpdir } = require('os')
+const path = require('path')
+const fs = require('fs')
+const { promisify } = require('util')
+const {
+  env: { DEPLOY_ID },
+} = require('process')
+
+const cpy = require('cpy')
+const { zipFunctions } = require('@netlify/zip-it-and-ship-it')
+const htmlToText = require('html-to-text')
+
+const isNetlifyCI = require('../build/src/utils/is-netlify-ci') // eslint-disable-line node/no-unpublished-require
```

Comment:
This should not work because `@netlify/build` is not available from this package. We probably should publish this as a separate utility. However the quick fix would be to simply inline that function (it's only one small expression).